absorption.cfg

```diff
@@ -144,14 +150,17 @@ mapmusic "sounds/music/track_06"
 // watercolour 0x103060
 // waterfallcolour 0x000000
 // waterfog 150
-// waterlod 3
+// waterlod 1
 // waterspec 150
 // watersubdiv 3
 // yawcloudlayer 0
 // yawclouds 0
 // yawenvlayer 0
 // yawsky 0
 
+gname = [editing]
+modetexs = [<grey>textures/modes/editing ]
+mut = [131072]
```

Comment:
These lines are from the loading screen interface and shouldn't be in the map file. I thought I fixed this problem but it still seems to happen.